bin/travis/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+function bloop_version {
+  cat build.sbt | grep "val bloop" | sed 's|[^0-9.]||g'
+}
+
+wget -O bin/coursier https://git.io/coursier-cli && chmod +x bin/coursier \
+&& bin/coursier launch ch.epfl.scala:bloopgun-core_2.12:$(bloop_version) -- server &>bloop.log &
+
+sbt "$1"
```

Comment:
Sbt command should be the last command - its status code signals if the tests passed or not